engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/preview/FacetLayerPreview.java

```diff
@@ -201,7 +201,7 @@ private static Rect2i worldToTileArea(Rect2i area) {
      */
     private BufferedImage rasterize(Region region) {
 
-        Vector3i extent = region.getRegion().size();
+        Vector3i extent = region.getRegion().getSize(new Vector3i());
         int width = extent.x;
         int height = extent.z;
```

Comment:
We can avoid the vector allocation here:
```suggestion
        int width =  region.getRegion().getSizeX();
        int height =  region.getRegion().getSizeZ();
```